support/ruby/collins-state/collins_state.gemspec

```diff
@@ -1,15 +1,11 @@
-# Generated by jeweler
-# DO NOT EDIT THIS FILE DIRECTLY
-# Instead, edit Jeweler::Tasks in Rakefile, and run 'rake gemspec'
 # -*- encoding: utf-8 -*-
 
 Gem::Specification.new do |s|
   s.name = "collins_state"
   s.version = File.read 'VERSION'
 
-  s.required_rubygems_version = Gem::Requirement.new(">= 0") if s.respond_to? :required_rubygems_version=
   s.authors = ["Blake Matheny"]
```

Comment:
@defect you could add yourself to the author's list if you wanted :smile: :cat2: :dizzy: